dash/testing/browser.py

```diff
@@ -213,11 +243,13 @@ def _get_firefox():
     def _is_windows():
         return sys.platform == "win32"
 
-    def multiple_click(self, css_selector, clicks):
+    def multiple_click(self, selector, clicks):
+        """multiple_clcik click the element with number of `clicks`"""
         for _ in range(clicks):
-            self.find_element(css_selector).click()
+            self.find_element(selector).click()
 
     def clear_input(self, elem):
+        """similate key press to clear the input with ActionChains"""
```

Comment:
simulate key presses to clear a text input

(I wouldn't bother mentioning `ActionChains`)